docs/root/start/building/building.rst

```diff
@@ -21,10 +21,12 @@ If the repo was not initially cloned recursively, you can manually initialize th
 Bazel requirements
 ------------------
 
-Envoy Mobile is compiled using Bazel 0.29.1.
+Envoy Mobile is compiled using the version of Bazel specified in the
+:repo:`.bazelversion <.bazelversion>` file.
 
-Take a look at our CI set up for :repo:`mac <ci/mac_ci_setup.sh>` and :repo:`linux <ci/linux_ci_setup.sh>`,
-in order to see how to install a specific Bazel version in your environment.
+Reviewing the CI setup for :repo:`Mac <ci/mac_ci_setup.sh>` and
```

Comment:
I believe `./bazelw` takes care of installing bazel. 

Do you mean for installing other requirements?

Reply:
Guessing that was probably the original intent of this sentence, but TBH I think it's confusing. Going to remove it instead.